packages/tc-schema-sdk/data-accessors/type.js

```diff
@@ -140,6 +145,7 @@ const createPropertiesWithRelationships = function ({
 					...updatedProps,
 					[propName]: {
 						...propDef,
+						...(propDef.cypher) && {otherNodeName: propDef.otherNodeName || propDef.type},   // only if its prop is a relationship
```

Comment:
I don't get why this should be here. This whole block only runs for non-relationships (cypher defined relationships are pseudo-non-relationships at this point as they don't need to deal with giving access to metadata on the relationship in the same way that ordinary relationship properties do)